src/node/blockstorage.cpp

```diff
@@ -361,19 +362,32 @@ void BlockManager::FindFilesToPruneManual(
         chain.GetRole(), last_block_can_prune, count);
 }
 
-uint64_t BlockManager::GetPruneTargetForChainstate(const Chainstate& chain, ChainstateManager& chainman) const
+uint64_t BlockManager::GetPruneTargetForChainstate(const Chainstate& chain, ChainstateManager& chainman, const uint64_t current_disk_usage) const
 {
     const auto number_of_chainstates{chainman.GetAll().size()};
     const uint64_t min_overall_target{MIN_DISK_SPACE_FOR_BLOCK_FILES * number_of_chainstates};
     auto target = std::max(min_overall_target, GetPruneTarget());
     uint64_t target_boost{0};
     if (m_opts.prune_target_during_init > -1 && chainman.IsInitialBlockDownload()) {
-        if ((uint64_t)m_opts.prune_target_during_init <= target) {
+        uint64_t prune_target_temporarily{(uint64_t)m_opts.prune_target_during_init};
+        if (prune_target_temporarily <= target) {
             target = std::max(min_overall_target, (uint64_t)m_opts.prune_target_during_init);
```

Comment:
prune_target_temporarily is now used in 2 contexts: first as "target during init" and then as "actually used target". for readability I would not mix them.

And in the context of "target during init": why not also use it in the std::max ?